packages/rest/src/agent.ts

```diff
@@ -0,0 +1,64 @@
+import type { TestLogger } from "./logger";
+import type { InitConfig } from "@aries-framework/core";
+
+import { Agent, HttpOutboundTransport, AutoAcceptCredential } from "@aries-framework/core";
+import { agentDependencies, HttpInboundTransport } from "@aries-framework/node";
+
+import { BCOVRIN_TEST_GENESIS } from "./util";
+
+export async function createAgent({
```

Comment:
I'd make this a test function (so only used in tests) and require the user of this library to provide his own agent instance. This allows for more flexibility in how you configure the agent.